apps/admin/frontend/src/components/adjudication_ballot_image_viewer.tsx

```diff
@@ -96,51 +96,44 @@ const BallotImageViewerControls = styled.div<{ isZoomedIn: boolean }>`
 export function BallotZoomImageViewer({
   imageUrl,
   ballotBounds,
-  writeInBounds,
+  zoomedInBounds,
 }: {
   imageUrl: string;
   ballotBounds: Rect;
-  writeInBounds: Rect;
+  zoomedInBounds: Rect;
 }): JSX.Element {
   const [isZoomedIn, setIsZoomedIn] = useState(true);
 
   const IMAGE_SCALE = 0.5; // The images are downscaled by 50% during CVR export, this is to adjust for that.
+
   const zoomedInScale =
-    (ballotBounds.width / writeInBounds.width) * IMAGE_SCALE;
+    (zoomedInBounds.width > zoomedInBounds.height
+      ? ballotBounds.width / zoomedInBounds.width
+      : ballotBounds.height / zoomedInBounds.height) * IMAGE_SCALE;
 
   return (
     <BallotImageViewerContainer>
       <BallotImageViewerControls isZoomedIn={isZoomedIn}>
         <Button
-          icon="ZoomOut"
-          onPress={() => setIsZoomedIn(false)}
-          disabled={!isZoomedIn}
-          color="neutral"
-          fill="tinted"
-        >
-          Zoom Out
-        </Button>
-        <Button
-          icon="ZoomIn"
-          onPress={() => setIsZoomedIn(true)}
-          disabled={isZoomedIn}
+          icon={isZoomedIn ? 'ZoomOut' : 'ZoomIn'}
+          onPress={() => setIsZoomedIn(!isZoomedIn)}
           color="neutral"
           fill="tinted"
         >
-          Zoom In
+          Zoom {isZoomedIn ? 'Out' : 'In'}
         </Button>
       </BallotImageViewerControls>
```

Comment:
Should we just make this be the same height as the header in the sidebar? It's close enough to make it feel misaligned.

![Screenshot-VxAdmin-2025-04-07T20:58:01 032Z](https://github.com/user-attachments/assets/d4b058e9-ab43-4cee-8c46-a746a947fc96)